app/.idea/.name

```diff
@@ -0,0 +1 @@
+ecosoft
```

Comment:
I don't think this file is needed, in fact it should be ignored by the git ignore. All .idea files should be